lib/util/esWrapper.js

```diff
@@ -92,7 +92,7 @@ const errorMessagesMapping = [
   },
   {
     // [index_not_found_exception] no such index, with { resource.type=index_or_alias & resource.id=foso & index=foso }
-    regex: /^no such index \[([%#])(.*)\/(.*)\]$/,
+    regex: /^no such index \[([%&])(.*)\.(.*)\]$/,
     subcode: 'index_or_collection_not_found',
     getPlaceholders: matches => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)